### DISCUSSION TITLE
Knowledge of coding

### DISCUSSION THREAD

**Discussion by Itz-Azlan** [community/community#66763] [Codespaces]
### Select Topic Area

General

### Body

[https://github.com/Itz-Azlan

**Comment by SomePogProgrammer** [comment]
### About this post

@Itz-Azlan 
In all due respect man, It's great that you want to share your repository, but why is it labeled codespaces, when you have no codespace even connected to this?
**It seems improperly labeled**, and I don't know why you made this post, but I'm sure it was a mistake, as you seem new to github. But I'd suggest editing the labels to "Show And Tell" and removing codespaces and general, this way the github community posts aren't improperly organized and those looking for something specific are able to find what they are looking for without off-topic posts! 

**Enjoy your journey with GitHub!**

Thanks, on behalf of our Community

- SomePogProgrammer
